cli/dts/lib.deno.ns.d.ts

```diff
@@ -250,6 +250,19 @@ declare namespace Deno {
   }
 
   export interface TestContext {
+    /**
+     * The current test name.
+     */
+    name: string;
+    /**
+     * File Uri of the current test code.
+     */
+    origin: string;
```

Comment:
Considering that `import.meta.url` is a `string`, I think it makes sense for this to be a string as well.